scripts/get_diag_string/get_diag_string.gml

```diff
@@ -0,0 +1,15 @@
+function get_diag_string(argument0, argument1, argument2, argument3, argument4, argument5) //gml_Script_get_diag_string
+{
+    with (obj_popup_dialogue)
+        instance_destroy()
+    var npd = instance_create(650, 326, obj_popup_dialogue)

```

Comment:
npd?

Reply:
newpopdialogue  I think, I changed it to this so it makes some sense